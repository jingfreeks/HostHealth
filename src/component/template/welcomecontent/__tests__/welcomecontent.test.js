import React from 'react';
import {render} from '@testing-library/react-native';
import WelcomeContent from '../welcomecontent';

describe('Welcome Button group Template Component', () => {
  it('Should work as expected to get snapshot', () => {
    const all = render(<WelcomeContent />);
    expect(all.toJSON()).toMatchSnapshot();
  });
});
