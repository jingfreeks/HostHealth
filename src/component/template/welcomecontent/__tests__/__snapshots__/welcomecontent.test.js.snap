// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome Button group Template Component Should work as expected to get snapshot 1`] = `
<RCTSafeAreaView
  style={
    [
      {},
    ]
  }
>
  <View
    alignType="center"
    style={
      [
        {
          "alignItems": "center",
          "alignSelf": "center",
          "flexDirection": "column",
        },
      ]
    }
  >
    <View
      style={
        [
          {
            "marginBottom": 58.85294117647059,
            "marginTop": 58.85294117647059,
          },
        ]
      }
    >
      <RCTSafeAreaView
        style={
          [
            {},
          ]
        }
      >
        <View
          alignType="center"
          style={
            [
              {
                "alignItems": "center",
                "alignSelf": "center",
                "flexDirection": "column",
              },
            ]
          }
        >
          <View
            style={
              [
                {
                  "marginBottom": 58.85294117647059,
                  "marginTop": 98.08823529411764,
                },
              ]
            }
          >
            <Image
              resizeMode="contain"
              source={
                {
                  "testUri": "../../../src/assets/mainlogo.png",
                }
              }
              style={
                [
                  {
                    "height": 196.17647058823528,
                    "width": 451.20588235294116,
                  },
                ]
              }
            />
          </View>
        </View>
      </RCTSafeAreaView>
      <View
        style={
          [
            {
              "alignItems": "center",
              "marginBottom": 20,
              "marginTop": 20,
            },
          ]
        }
      >
        <Text
          style={
            [
              {
                "color": "#0e3f6c",
                "fontFamily": "Avenir",
                "fontSize": 18,
                "fontWeight": "900",
              },
            ]
          }
        />
        <Text
          style={
            [
              {
                "color": "#0e3f6c",
                "fontFamily": "Avenir",
                "fontSize": 16,
                "fontWeight": "400",
              },
            ]
          }
        />
      </View>
    </View>
  </View>
</RCTSafeAreaView>
`;
